config.json

```diff
@@ -161,6 +161,24 @@
         "lp": []
       },
       "hash": "9915255fd06766994bf53a45f81584ad"
+    },
+    {
+      "id": "origami",
+      "name": "Origami",
+      "icon": "logo.png",
+      "metadata": {
+        "pt": [
+          {
+            "chainId": 1,
+            "address": "0xbbf399db59a845066aafce9ae55e68c505fa97b7",
+            "integrationUrl": "https://origami.finance/vaults/1-0xb9dad3693AEAc9025Cb24a47AFA6930539877187/info",
+            "description": "A 5x leveraged position on PT sUSDe OCT 2024"
+          }
+        ],
+        "yt": [],
+        "lp": []
+      },
+      "hash": "491aa773ce939c7df3c0a3467a8fd8a8"
```

Comment:
Hi, you don't need to update the global config file, it will be updated via CI. Thanks!